src/Aspire.Dashboard/Utils/FormatHelpers.cs

```diff
@@ -58,16 +77,16 @@ public static string FormatTimeWithOptionalDate(DateTime value, bool includeMill
         {
             // e.g. "08:57:44" (based on user's culture and preferences)
             // Don't include milliseconds as resource server returned time stamp is second precision.
-            return FormatTime(local, includeMilliseconds);
+            return FormatTime(local, includeMilliseconds, provider);
         }
         else
         {
             // e.g. "9/02/2024 08:57:44" (based on user's culture and preferences)
-            return FormatDateTime(local, includeMilliseconds);
+            return FormatDateTime(local, includeMilliseconds, provider);
         }
     }
 
-    public static string FormatNumberWithOptionalDecimalPlaces(double value, IFormatProvider? provider = null)
+    public static string FormatNumberWithOptionalDecimalPlaces(double value, CultureInfo? provider = null)
```

Comment:
Consider renaming `provider` to `cultureInfo` here and elsewhere. "Provider" is a very generic term.

Would also be nice to doc what happens if you pass null (here and elsewhere).

Reply:
The behavior is the same as other format methods. No culture = CurrentCulture.